File: JavaReactKampHomeWork/homeworkJavaSE1.8June/src/day4Homework3GamerBackEnd/concretes/GamerCheckManager.java
package day4Homework3GamerBackEnd.concretes;

import day4Homework3GamerBackEnd.abstracts.GamerCheckService;
import day4Homework3GamerBackEnd.entities.Gamer;

public class GamerCheckManager implements GamerCheckService{

	@Override
	public boolean checkIfRealPerson(Gamer gamer) {
		return true;
	}

}

File: YazilimGelistiriciDotNetHomeWork/source/repos/KampHomeWorkJune/Day5Homework5GamerBackEnd/Concretes/GamerCheckManager.cs
using Day5Homework5GamerBackEnd.Abstracts;
using System;
using System.Collections.Generic;
using System.Text;

namespace Day5Homework5GamerBackEnd.Concretes
{
    class GamerCheckManager : IGamerCheckService
    {
        public bool CheckIfRealPerson(Gamer gamer)
        {
            return true;
        }
    }
}
